### DISCUSSION TITLE
discord server?

### DISCUSSION THREAD

**Discussion by chirrupted** [justchokingaround/lobster#118] [Ideas]
it would be cool if there was a discord server maybe (i.e for community support and api status and such)

**Comment by justchokingaround** [comment]
atm i'm very active in the ani-cli discord server (i'm the owner), but if there are more people that would like to have a separate lobster server, with a flixhq rss feed, and for general help, discussion etc. i would be totally down

**Reply by chirrupted** [reply]
that'd be really cool! 🙏🙏

**Reply by justchokingaround** [reply]
https://discord.gg/JTDS2CKjQU
